### ISSUE TITLE
no required module provides package

### ISSUE THREAD

**Issue by smingins** [willbryant/homebrew-verm#5]
I am trying to install and getting the following error:

```
go
build
-ldflags
-X main.compiled_version=0.52 -X main.compiled_root_data_directory=/usr/local/var/verm
github.com/willbryant/verm

no required module provides package github.com/willbryant/verm: working directory is not part of a module

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/willbryant/homebrew-verm/issues
```

I don't know go ... google seems to suggest that is the root cause ... just wondering if you might know off the top of your head?

P.S.  Hi @willbryant :-)

**Comment by smingins** [comment]
I did some hackery ... installed go 1.15 (instead of 1.16) ... and installed without dependencies ... and seemed to work,

**Comment by mpluhar** [comment]
Same issue here.  Adding this line to verm.rb fixes the build for me when using go 1.16
```
ENV["GO111MODULE"] = "auto"
```

I would be interested in a proper solution as this works only until go 1.17 is around.

**Comment by willbryant** [comment]
Thanks both and hi @smingins :). I have to confess I'd forgetton all about this tap so it was installing a very very old version, but fixed now.
